drivers/ethernet/eth_dw.c

```diff
@@ -136,7 +136,7 @@ static void eth_tx_spin_wait(struct eth_runtime *context)
 
 static void eth_tx_data(struct eth_runtime *context, u8_t *data, u16_t len)
 {
-#ifdef CONFIG_NET_DEBUG_L2_ETHERNET
+#ifdef CONFIG_ETHERNET_LOG_LEVEL >= LOG_LEVEL_DBG
```

Comment:
Shouldn't this be written as `#if CONFIG_ETHERNET_LOG_LEVEL >= LOG_LEVEL_DBG` (so not with #ifdef)

Reply:
you are right redone it